functions/go/set-namespace/main.go

```diff
@@ -1,75 +1,139 @@
+// This file will be processed and embedded to pluginator.
```

Comment:
where does unit tests for this logic live?

Reply:
The main functions are tested in the pluginator. But I will add a test for this *function* as well.